<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>南水北调智能视频监控</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/images/nsbd.ico</normaloff>:/resources/images/nsbd.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_showFrame">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>711</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(71, 71, 71);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_showCover">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>88</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>显示cover</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_rule">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>731</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(-1,-1,-1);
selection-background-color: rgb(-1,-1,-1);
</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_rule">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>-1</y>
      <width>20</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(-1,-1,-1);
selection-background-color: rgb(-1,-1,-1);</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_waterLevelValue">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>230</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>单位长度：</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_unitPix">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>0</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>600</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>90</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1145</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="separator"/>
   <addaction name="action_run"/>
   <addaction name="action_stop"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/document-open.png</normaloff>:/resources/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开视频文件</string>
   </property>
  </action>
  <action name="action_run">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/media-playback-start-symbolic.svg</normaloff>:/resources/images/media-playback-start-symbolic.svg</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="toolTip">
    <string>运行</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/media-playback-pause-symbolic.svg</normaloff>:/resources/images/media-playback-pause-symbolic.svg</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>停止</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
